src/ui/controllers/course/index.ts

```diff
@@ -180,6 +180,7 @@ export async function loadCourse(
 	const course = await catalog.get(courseId)
 	if (course) {
 		req.course = course
+		console.log(course)
```

Comment:
Remove or change to log4js with meaningful message.

Reply:
Removed